Realm/Tests/RLMMultiProcessTestCase.m

```diff
@@ -77,6 +75,7 @@ - (NSTask *)childTask {
     NSString *testName = [NSString stringWithFormat:@"%@/%@", self.className, self.testName];
     NSMutableDictionary *env = [NSProcessInfo.processInfo.environment mutableCopy];
     env[@"RLMProcessIsChild"] = @"true";
+    [env removeObjectForKey:@"XCTestConfigurationFilePath"];
```

Comment:
Xcode 7 switched from passing all the undocumented options for things like how to communicate with the IDE from the command line to a config file, with the path specific in the env variable. We don't want the child process to use these settings because multiple xctest processes communicating with the IDE doesn't work at all.